kimchi_navigation/launch/kimchi_nav_and_slam.launch.py

```diff
@@ -30,7 +30,7 @@ def generate_launch_description():
 
     map_argunment = DeclareLaunchArgument(
         "map",
-        default_value="",
+        default_value=os.path.join(pkg_kimchi_nav, "maps/hq_map/map.yaml"),
```

Comment:
This line was added for testing. Please remove it as it might break the current implementation